### ISSUE TITLE
Marketing

### ISSUE THREAD

**Issue by Sarke** [adoptoposs/adoptoposs#92]
I only found it because I had the same idea, otherwise I wouldn't have known it existed.

It is a fantastically useful project, but if no one knows about it then it's not.  In your blog post mentioned:

> After I found that, on GitHub alone, there were more than 36,000 issues asking "Is this project abandoned?", I thought about how to tackle this problem. More than 15,000 of these were open issues. So, lots of projects need help with their maintenance.

So, how do we connect those projects with this one?

I would suggest a nomination feature, where people can nominate a project, and then others can vote on it to give it more visibility.  I realize you already thought of this:

> First, I thought it might be nice to have a dashboard that rates OSS projects by whether they needed help with their maintenance. When I had a prototype running after a couple of days' work, I found that it kind of worked but did not really overcome the problem of getting into contact with the maintainer. And when a maintainer was hard to reach when I needed support, it would already be too late.

So how are we getting in contact with the maintainers now?

A nomination feature could put some visible pressure on the current inactive maintainers (e.g. "look, many people want this project to stay alive.").  I suspect many of the inactive maintainers aren't even thinking about those projects anymore, so expecting them to come here (without even knowing this exists) and submit their project(s) without a little nudge is unrealistic.

Maybe even have a notify feature, when enough votes have been cast for a project you can either tweet at the person (if their github has their twitter listed) with a link to the project nomination on adoptoposs.org.  Another way would be to have a separate github repo for just issues that are autogenerated and tagged with the person's github handle.

---

There are other ways to get this out there.  Maybe contacting large multi-part projects like GNOME, KDE, MATE, etc, that have many smaller parts that lack maintainers.  Those projects will most likely know already which one of their smaller programs are in need of maintainers.

Another way would be to start with those 15,000 issues mentioned above, sort them by number of github stars, and then contact the top 100 or so manually.

---

Another feature that could be useful would be to try to lure the currently inactive maintainers back.  Combined with the nomination feature, you could have a sort of pledge system where people can pledge to support the project financially (e.g. through Github Sponsors, Open Collective, Patreon, etc.) if they start maintaining the project again.

Obviously there could be some issues with people following through with those pledges, so those kinks would need to be worked out.

Perhaps it would be enough to get the maintainers interested again.

---

I also think the github stars and date submitted should be listed on the projects page.

Also, add the website name to the README.  I know it's listed in the description, but it should also be prominently displayed in the README.

Anyways, that's my 2 cents worth.

**Comment by paulgoetze** [comment]
Hey @Sarke, thanks for opening the issue and the detailed descriptions of your ideas and very helpful hints. 
Happy you found the project :)

I think you have a good point here. Unless there's maintainers submitting their project and other developers who know that Adoptoposs is a place to go to and find projects to help maintain, it's of no real use. On top, that group of people would need to be willing to help maintain OSS projects. We are very aware of the critical-mass problem and of course we are openly looking for ways to overcome it.

**Regarding the questions you brought up:** 

> So, how do we connect those projects with this one?

The initial goal of Adoptoposs was to even prevent situations like the mentioned 15k "Is this abandonned?" issues. One goal is to educate and help active maintainers to give other people access to their projects and have co-maintainers. Ideally, all that should happen before they cannot/don't want to maintain the project anymore. In my opinion this is more sustainable in the long run, than keeping a single-maintainer open-source world, where people burn out over the load of work and projects are only passed on to someone in the community after the project was basically already abandonned for a while. 

That's why I thought it might be good to focus on active maintainers first, when I started building the projects 6 months ago. I just did not consider these mentioned 15k possibly already abandonned ones until recently. (Btw, in the blog post I meant an automated project rating approach, I did not try any rating by users back then).

However, I hear what you say, and I more and more like the idea of a nomination features. We have an issue opened by @abathur with a similar proposal, and I'd be happy to move the conversation about it to #79 and get into more details there.

---

> So how are we getting in contact with the maintainers now?

Right now it's very much based on directly addressing maintainers. We have a Twitter account and regularily do a search and message people who are looking for new (co-)maintainers. We did some talks on tech-meetups (COVID-19 does not make it easier really, to get into contact with potentially interested people) and commented on GitHub repos. There are the blog posts you probably also found, and I'm currently working on some more posts to make people aware of the maintainenance problem.

All in all it's pretty much word-of-mouth advertising right now. 

We can certainly improve the project's visibility in search engines and expedite spreading the word by better targeting relevant groups (e.g. in programming mailing lists, newsletters, forums, etc.).

I really like your idea of contacting larger multi-part projects. I'm not a marketing expert at all, and if you have more ideas how to make the project better known, and you maybe even have time and connections to do so, we'd really appreciate all kind of help here. Do you have connections to/Would you be up for helping us contact some of the mentioned projects?

---

Regarding your idea to lure back inactive maintainers: 

I'm not sure whether that would easily work. There's all kinds of reasons why people cease putting time into their projects and they stop working on it for a reason. I guess sometimes it is the money and/or time-consuming aspect. From what I found, I would consider contacting inactive maintainers (and transfering the project knowledge to more people) the harder part, than actually finding people to take over a widely used project. 

But of course that's my view, and I would like to go into further dicussion of such a feature.

---

> I also think the github stars and date submitted should be listed on the projects page.

* What would be the criteria for you to consider helping a project? 
* Would that be the same for e.g. a friend of yours? 

That's not a "no they shouln't", I just try to figure out, how and why people would be attracted by a certain project, so please let me know your thoughts behind this.

> Also, add the website name to the README.

Now that you mention it, I think that's a good idea. Feel free to open a pull request with the change.

--- 

In general, which of the ideas and points you described do you feel are the most important ones?
And to what extend would you be up to help putting them into reality?
